frontend/testing/requests/authorized-users.test.js

```
@@ -5,6 +5,8 @@ const {
   fetchContentCreators,
   fetchWebsiteCreators,
   getAllAuthorizedUsers,
+  createAuthorizedUser,
+  updateUserInfo,
```

ESLint

[error] 'updateUserInfo' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)